keps/sig-node/2254-cgroup-v2/kep.yaml

```diff
@@ -21,5 +21,5 @@ see-also:
 replaces:
 superseded-by:
 
-latest-milestone: "0.0"
+latest-milestone: "v1.19"
 stage: "alpha"
```

Comment:
Latest milestone should be 1.22 / stage beta, right?

Reply:
correct.

I'll fix it, thanks for the review!